chips/veer_el2/src/pic.rs

```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 // Copyright Tock Contributors 2022.
 
-//! Platform Level Interrupt Control peripheral driver for SweRV EH1.
+//! Platform Level Interrupt Control peripheral driver for VeeR.
```

Comment:
There are no interrupts generated right? So this isn't tested.

Might be worth commenting that somewhere encase it turns out to be different to the original

Reply:
Right, I've added a comment below this line to explain this case.